### DISCUSSION TITLE
graalvm minio DownloadObjectArgs must have no argument constructor

### DISCUSSION THREAD

**Discussion by maomaochong2dai** [oracle/graal#5871] [Q&A]
code:
 public  void download(String  bucketName,String minioFileName,String fileName) {
        minioClient.downloadObject(
                DownloadObjectArgs
                        .builder()
                        .bucket(bucketName)
                        .object(minioFileName)
                        .filename(fileName)
                        .build()
        );
    }
idea no promble,but exe has promble:
Caused by: java.lang.RuntimeException: class io.minio.DownloadObjectArgs must have no argument constructor
        at io.minio.BaseArgs$Builder.newInstance(BaseArgs.java:135) 
        at io.minio.BaseArgs$Builder.build(BaseArgs.java:147)

environment:
graalvm22.3.1  jdk17   springboot3.0  minio8.5.1  when i  mvn -Pnative native:compile   get    .exe

**Comment by alina-yur** [comment]
I would assume this has to be an issue rather than a discussion?
